src/views/v2/month/top-bar/nav/prev.php

```diff
@@ -22,6 +22,9 @@ class="tribe-common-c-btn-icon tribe-common-c-btn-icon--caret-left tribe-events-
 		aria-label="<?php esc_attr_e( 'Previous month', 'the-events-calendar' ); ?>"
 		title="<?php esc_attr_e( 'Previous month', 'the-events-calendar' ); ?>"
 		data-js="tribe-events-view-link"
+		<?php if ( ! $skip_empty && $prev_month !== $prev_event_date->format( 'n' )  ) : ?>
+			rel="nofollow noindex"
+		<?php endif; ?>
```

Comment:
🤔

Reply:
I did this: `rel="<?php echo esc_attr( $prev_rel ); ?>"` with the added caveat that instead of an empty value it will pass "prev" or "next" https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#attr-next